utils/scurveAlgos.py

```diff
@@ -478,6 +478,8 @@ def anaUltraScurve(args, scurveFilename, calFile=None, GEBtype="short", outputDi
         scurveFitTree.Branch( 'pedestal', pedestal, 'pedestal/F')
         ped_eff = array( 'f', [ 0 ] )
         scurveFitTree.Branch( 'ped_eff', ped_eff, 'ped_eff/F')
+        nevents = array( 'i', [ 0 ] )
+        scurveFitTree.Branch( 'nevents', nevents, 'nevents/I')
```

Comment:
Adding a new branch, which increases memory for the output file unnecessarily (e.g. because `nevents` will be the same for entries in the `TTree`) it's probably better to simply change `ped_eff` to store a percentage rather than an integer.  The pro's and cons of this should have been discussed in the issue.

Reply:
`ped_eff` is redefined as `ped_eff/nevents` now